### ISSUE TITLE
"Found invalid data while decoding" .nut file

### ISSUE THREAD

**Issue by ccdesu** [arcusmaximus/VNTranslationTools#57]
I'm trying to extract the script files for the Vita version of [Kamiyaba](https://vndb.org/v19769) but when I try to extract the .nut files, I get "Found invalid data while decoding." The errors persists even after rebuilding VNTextPatch with the encoding set to Shift_JIS which seems to be what the script is encoded in. Here are the files for reference [script.zip](https://github.com/arcusmaximus/VNTranslationTools/files/8971941/script.zip)
Thanks!

**Comment by Cosetto** [comment]
> I'm trying to extract the script files for the Vita version of [Kamiyaba](https://vndb.org/v19769) but when I try to extract the .nut files, I get "Found invalid data while decoding." The errors persists even after rebuilding VNTextPatch with the encoding set to Shift_JIS which seems to be what the script is encoded in. Here are the files for reference [script.zip](https://github.com/arcusmaximus/VNTranslationTools/files/8971941/script.zip) Thanks!

I don't think this is the script of Mware engine. Mware engine should be script.npk and script header is **PRCS**. Furthermore, Mware is Nitro+ engine, why is there another brand using it lol
